python/pyarrow/tests/test_compute.py

```diff
@@ -1634,6 +1634,10 @@ def test_is_null():
     expected = pa.array([False, False, False, True, True])
     assert result.equals(expected)
 
+    result = arr.is_nan()
+    expected = pa.array([False, False, False, False, True])
```

Comment:
```suggestion
    expected = pa.array([False, False, False, None, True])
```

Reply:
Thanks for the suggestion, fixed this in a commit together with updating the docstring 👍🏻